Python/PdfReader/pdf_reader.py

```diff
@@ -0,0 +1,23 @@
+import pyttsx3 #python text to speech version 3
+import PyPDF2 
+
+pdf= open("Andriy_Burkov.pdf",'rb') #you can replace this with any pdf file 
+
+pdfreader = PyPDF2.PdfFileReader(pdf)
+
+pages=pdfreader.numPages 
+print(pages)
+
+
+dell= pyttsx3.init() #intialising object 
+
+pages=pdfreader.getPage(7)
+
+text=pages.extractText() #extract text from pages 
```

Comment:
Do not write comments beside the code, shift them above the code to maintain the coding standard. Its advised to practice writing python code according to PEP8 rules, this will help you to write cleaner code in long run.